plugin/CactbotEventSource/FFXIVProcessIntl.cs

```diff
@@ -698,5 +700,33 @@ public struct SamuraiJobMemory {
         }
       }
     }
+
+    [StructLayout(LayoutKind.Explicit)]
+    public struct SageJobMemory {
+      [FieldOffset(0x00)]
+      public ushort addersgallGauge_bits; // the addersgall gauge, from 0 to 19999.
+
+      [FieldOffset(0x02)]
+      public byte addersgall;
+
+      [FieldOffset(0x03)]
+      public byte addersting;
+
+      [FieldOffset(0x04)]
+      private byte eukrasia_bits;
+
+      public int addersgallGauge {
+        get {
+          // translate the bits to a value from 0 to 100.
+          return (int)((float)addersgallGauge_bits / 20000 * 100);
```

Comment:
Do you think it's worth sending the whole value, rather than truncating here?

Reply:
you mean the original value from memory?

Reply:
well, it can be also converted in js side, I don't care where it is. sure

Reply:
Yeah, that sounds good!